packages/ndla-editor/src/structure/Structure.js

```diff
@@ -94,6 +96,12 @@ const Structure = ({
               const greyedOut = highlightMainActive
                 ? !isNewMainActive && !isMainActive && openedPaths.length > 0
                 : !isOpen && isSubject && openedPaths.length > 0;
+
+              const isVisible =
+                metadata !== undefined && metadata.visible
+                  ? metadata.visible
+                  : false;
```

Comment:
Kunne det fungert med chaining her? Eg trur også at dersom metadata ikkje er definert skal vi vise alt slik at defaulten er true.
```suggestion
              const isVisible = metadata?.visible || true;
```

Reply:
Det vil bare returnere true i alle tilfeller. 

La inn dette fordi jeg ikke vet om Structure blir brukt noe steder hvor man ikke har inkludert includeMetadata=true i api-kallet. Vet ikke om det er sansynlig?

Reply:
Det er sant det tenkte eg ikkje på.